tests/testthat/test_calc.dtsurvey.R

```diff
@@ -410,3 +411,93 @@ test_that('Where elimiates everything', {
 
 })
 
+test_that('MI approach', {
+
+  midat = lapply(1:10, function(x){
+    r = apiclus1
+    r$random = sample(1:5, nrow(r), T)
+    r$random3 = sample(1:3, nrow(r), T)
+    r$random_fact = factor(r$random)
+    dtsurvey(r, 'dnum', weight = 'pw')
+
+  })
+
+  midat = mitools::imputationList(midat)
+  misur = svydesign(id=~dnum, weights=~pw, data=midat)
+
+  # Numeric variable unrelated
```

Comment:
More commenting could be useful. could be as simple as noting r1.1 is testing the use of mitools directly. r1.2 is testing the passing of an mitools:: imputationList to calc(), and r1.3 is passing the contents of the first object in the imputationlist as a dtsurvey/data.table object.